File: Examples/VisualBasic/Files/Utility/SetPDFCreationTime.vb
Imports System
Imports Aspose.Cells
Imports Aspose.Cells.Charts

Namespace Files.Utility
    Friend Class SetPDFCreationTime
        Public Shared Sub Run()

            ' ExStart:1
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)
            Dim inputPath As String = dataDir & "Book1.xlsx"
            ' Load excel file containing charts
            Dim workbook As New Workbook(inputPath)

            ' Create an instance of PdfSaveOptions and pass SaveFormat to the constructor
            Dim options As New PdfSaveOptions(SaveFormat.Pdf)
            options.CreatedTime = Date.Now

            ' Save the workbook to PDF format while passing the object of PdfSaveOptions
            workbook.Save(dataDir & "output.pdf", options)
            ' ExEnd:1

        End Sub
    End Class
End Namespace

File: Examples/CSharp/Files/Utility/SetPDFCreationTime.cs
using System;
using Aspose.Cells;
using Aspose.Cells.Charts;

namespace Aspose.Cells.Examples.CSharp.Files.Utility
{
    class SetPDFCreationTime
    {
        public static void Run()
        {

            // ExStart:1
            string dataDir = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);
            string inputPath = dataDir + "Book1.xlsx";
            // Load excel file containing charts
            Workbook workbook = new Workbook(inputPath);

            // Create an instance of PdfSaveOptions and pass SaveFormat to the constructor
            PdfSaveOptions options = new PdfSaveOptions(SaveFormat.Pdf);
            options.CreatedTime = DateTime.Now;

            // Save the workbook to PDF format while passing the object of PdfSaveOptions
            workbook.Save(dataDir + "output.pdf", options);
            // ExEnd:1

        }
    }
}
